### PR TITLE
add `monitor/schema` action

### PR DESCRIPTION
**Pull request by ronenlu**
example [run](https://github.com/ariga/atlas-action/actions/runs/11910339752/job/33189530687?pr=256#step:4:119):

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -447,6 +448,29 @@ Approve a declarative migration plan.
 * `link` - The URL of the plan in [Atlas Registry](https://atlasgo.io/registry).
 * `status` - The status of the plan. For example, `PENDING` or `APPROVED`.
 
+### `ariga/atlas-action/agent`
+
+Run an [Atlas Agent](https://atlasgo.io/cloud/agents) that execute cloud jobs on your database.
```
Comment (noamcattan) [inline README.md:453]: This sentance is not clear enough about what the action is doing, how it works etc.

File: agent/entrypoint.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#!/bin/sh -l
+
+set -e
+
+echo "Running Atlas Agent"
+/atlas-agent --token "$CLOUD_TOKEN" &
+sleep "$WAIT_TIME"
```
Comment (masseelch) [inline agent/entrypoint.sh:7]: This means that if an agent gets a job on second 29 it only has a second left to esecute the job before being killed forcefully.

We need something more graceful.

File: README.md [hunk 3]
```diff
@@ -447,6 +448,37 @@ Approve a declarative migration plan.
...
+
+Takes snapshot of the database schema and uploads it to Atlas Cloud.
+Can be used periodically to [monitor](https://atlasgo.io/monitoring) changes in the database schema.
+
+#### Inputs
+
+* `cloud-token` - (required) The token that is used to connect to Atlas Cloud (should be passed as a secret).
+* `url` - (required) The URL of the database to take snapshot of. For example: `mysql://root:pass@localhost:3306/prod`.
+* `extID` - (optional) Unique identifier for the database server.
+* `schemas` - (optional) List of database schemas to include in snapshot (by default includes all schemas). see: https://atlasgo.io/declarative/inspect#inspect-multiple-schemas
```
Comment (masseelch) [inline README.md:461]:
```suggestion
* `schemas` - (optional) List of database schemas to include in snapshot (by default includes all schemas). see: https://atlasgo.io/declarative/inspect#inspect-multiple-schemas.
```

File: README.md [hunk 4]
```diff
@@ -447,6 +448,37 @@ Approve a declarative migration plan.
...
+* `exclude` - (optional) List of exclude patterns from snapshot inspection. see: https://atlasgo.io/declarative/inspect#exclude-schemas
```
Comment (masseelch) [inline README.md:462]:
```suggestion
* `exclude` - (optional) List of exclude patterns from snapshot inspection. see: https://atlasgo.io/declarative/inspect#exclude-schemas.
```

File: README.md [hunk 5]
```diff
@@ -447,6 +448,37 @@ Approve a declarative migration plan.
...
+If the URL of the database changes, atlas knows to track that is the same database via the `extID` parameter.
+
+```yaml
+        uses: ariga/atlas-action/monitoring@v1
+        with:
+          cloud-token: ${{ secrets.ATLAS_CLOUD_TOKEN }}
+          url: 'mysql://root:pass@localhost:3306'
+          schemas: 'dev,users'
+          extID: 'dev-db'
+```
```
Comment (masseelch) [inline README.md:480]: Indention is weird here.

Comment (ronenlu) [reply README.md:480]: this is the right Indention for step in github workflow yaml (so user can copy and paste)

**Review by masseelch — COMMENTED** [review]
LGTM
@a8m @rotemtam I'll let one of you approve.

File: README.md [hunk 6]
```diff
@@ -447,6 +448,37 @@ Approve a declarative migration plan.
...
+
+Sync the database schema to Atlas Cloud.
+Can be used periodically to [monitor](https://atlasgo.io/monitoring) changes in the database schema.
+
+#### Inputs
+
+* `cloud-token` - (required) The token that is used to connect to Atlas Cloud (should be passed as a secret).
+* `url` - (required) The URL of the database to sync. For example: `mysql://root:pass@localhost:3306/prod`.
+* `slug` - (optional) Unique identifier for the database server.
+* `schemas` - (optional) List of database schemas to include (by default includes all schemas). see: https://atlasgo.io/declarative/inspect#inspect-multiple-schemas.
```
Comment (rotemtam) [inline README.md:461]: add example (its not obvious that its `schema_a,schema_b`

File: monitor/schema/action.yml [hunk 7]
```diff
@@ -0,0 +1,36 @@
+name: 'Atlas Schema Monitor'
```
Comment (a8m) [inline monitor/schema/action.yml:1]:
```suggestion
name: 'Atlas Schema Monitoring'
```

**Pull request closed without merging**